src/z3c/autoinclude/dependency.txt

```diff
@@ -86,8 +86,8 @@ that it depends on that have ``configure.zcml`` or ``meta.zcml``.
 Note that the individual lists within ``includableInfo`` preserve the
 package order defined in ``setup.py``::
 
-    >>> a_include_finder.includableInfo(['configure.zcml', 'meta.zcml'])
-    {'configure.zcml': ['b.c'], 'meta.zcml': ['z3c.autoinclude', 'testdirective']}
+    >>> list(sorted(list(a_include_finder.includableInfo(['configure.zcml', 'meta.zcml']).items())))
```

Comment:
pprint() sorts the dictionary, unless you're on python 2.4 and len(repr(the_dictionary)) < 70 or something like that.  zope.testing used to have a fork of pprint.pprint to fix that one little buglet.

The pprint changes in Python 3.4 are new to me and sound like a valid reason not to use pprint (or doctests). ;)